user_docs/en/changes.t2t

```diff
@@ -30,6 +30,10 @@ What's New in NVDA
     -
   - Added a new unassigned input gesture to toggle the reporting of figures and captions. (#10826, #14349)
   -
+- A new braille mode called "display speech output" has been added. (#15898, @Emil-18)
+  - When active, the braille display shows exactly what NVDA is speaking.

```

Comment:
Given this would display paused speech, cancelled speech and when speech mode is off this may be more accurate

```suggestion
  - When active, the braille display shows exactly what NVDA prepares to speak.
```